datalad/interface/common_opts.py

```diff
@@ -148,4 +148,14 @@
     [PY: Multiple dependencies can be given as a list of sibling names
     PY][CMD: This option can be given more than once to configure multiple
     dependencies CMD]""",
+    action='append',
+    constraints=EnsureStr() | EnsureNone())
+
+publish_by_default = Parameter(
+    args=("--publish-by-default",),
+    metavar='REFSPEC',
+    doc="""add a refspec to be published to this sibling by default if nothing
+    specified.""",
+    constraints=EnsureStr() | EnsureNone(),
     action='append')
```

Comment:
`--publish-by-default=refs/tags/* --publish-by-default=refs/heads/*` should work, yes.
Did you notice `action='append'` or do I get sth wrong?